Makefile

```diff
@@ -85,6 +85,9 @@ vendor: bin/dep ## Install dependencies
 config/config.toml:
 	cp config/config.toml.example config/config.toml
 
+config/dex-config.yml:
```

Comment:
If the name `dex-config.yml` is not mandatory, I would just call it `dex.yml`

Reply:
renamed, thanks